frontend/public/components/default-resource.jsx

```diff
@@ -25,75 +27,21 @@ const tableColumnClasses = [
   Kebab.columnClass,
 ];
 
-const TableHeader = () => {
-  return [
-    {
-      title: 'Name',
-      sortField: 'metadata.name',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[0] },
-    },
-    {
-      title: 'Namespace',
-      sortField: 'metadata.namespace',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[1] },
-    },
-    {
-      title: 'Created',
-      sortField: 'metadata.creationTimestamp',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[2] },
-    },
-    {
-      title: '',
-      props: { className: tableColumnClasses[3] },
-    },
-  ];
-};
-TableHeader.displayName = 'TableHeader';
-
-const TableRowForKind = ({ obj, index, key, style, customData }) => {
-  const kind = referenceFor(obj) || customData.kind;
-  const menuActions = [...Kebab.getExtensionsActionsForKind(kindObj(kind)), ...common];
-
-  return (
-    <TableRow id={obj.metadata.uid} index={index} trKey={key} style={style}>
-      <TableData className={tableColumnClasses[0]}>
-        <ResourceLink
-          kind={customData.kind}
-          name={obj.metadata.name}
-          namespace={obj.metadata.namespace}
-          title={obj.metadata.name}
-        />
-      </TableData>
-      <TableData className={classNames(tableColumnClasses[1], 'co-break-word')}>
-        {obj.metadata.namespace ? (
-          <ResourceLink
-            kind="Namespace"
-            name={obj.metadata.namespace}
-            title={obj.metadata.namespace}
-          />
-        ) : (
-          'None'
-        )}
-      </TableData>
-      <TableData className={tableColumnClasses[2]}>
-        <Timestamp timestamp={obj.metadata.creationTimestamp} />
-      </TableData>
-      <TableData className={tableColumnClasses[3]}>
-        <ResourceKebab actions={menuActions} kind={kind} resource={obj} />
-      </TableData>
-    </TableRow>
-  );
-};
-
 export const DetailsForKind = (kind) =>
   function DetailsForKind_({ obj }) {
+    const { t } = useTranslation();
+
+    const getKindLabel = (item) => {
+      const model = modelFor(item);
+      return model?.labelKey || kindForReference(item);
```

Comment:
Sorry, my code suggestion was wrong.

```suggestion
      return t(model?.labelKey) || kindForReference(item);
```

Reply:
I should have noticed. :) Fixed.